resources/js/Pages/Welcome.tsx

```diff
@@ -99,7 +114,7 @@ export default function Welcome({ auth }: PageProps) {
                                     <time className="font-mono italic text-secondary">
                                         1998 - 2017
                                     </time>
-                                    <div className="text-lg font-black text-white">
+                                    <div className="text-lg font-black text-gray-700">
```

Comment:
So daiseyUI allows us to set a theme, and then the colors chosen for the theme are available as a set of options for the tailwind components, such as`text-accent`, `text-error`, `btn-primary`, `btn-secondary`, etc.. 

So can these be changed to `text-neutral` or `text-primary` or whatever is closest to the gray you chose?

Reply:
Got it.  Thanks for the feedback :thumbsup: